.github/workflows/test_github_app _target_org.yml

```diff
@@ -0,0 +1,54 @@
+name: test-github_app
```

Comment:
can you remove this test out of the PR? Let's put this as a new feature into another PR 👾

Reply:
done